src/onemod/fsutils/interface.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path
+from typing import Any, Literal
+
+import polars as pl
+
+from onemod.fsutils.config_loader import ConfigLoader
+from onemod.fsutils.data_loader import DataLoader
+from onemod.fsutils.path_manager import PathManager
+
+
+class DataInterface(PathManager):
```

Comment:
I am definitely ok to change this to `FileInterface` to reduce the probability for confusion.

Reply:
If this is in response to my naming-related mutterings, I really don't mind too much ^^; I still like the ring of `DataInterface`, and frankly `dataif`, better, so if you don't mind I actually prefer not to change this for now.

However happy to change it if you feel more strongly about this.

Reply:
That sounds good!